cg/services/order_validation_service/workflows/mutant/constants.py

```diff
@@ -4,7 +4,7 @@
 
 
 class MutantDeliveryType(Enum):
-    ANALYSIS_FILES = DataDelivery.FASTQ_ANALYSIS
+    ANALYSIS_FILES = DataDelivery.FASTQ
```

Comment:
This has to be changed as @ChrOertlin fixed this

Reply:
Has that been merged? As far as I know, we are still keeping them as fastq. Could we create an issue and revisit this when it has gone live?

Reply:
It is listed in https://github.com/Clinical-Genomics/improve-order-flow/issues/99